LEARN.md

```diff
@@ -0,0 +1,164 @@
+# Learn Pulumi
+
+So you're interested in writing some Learn Pulumi content? Great! Follow these
+steps to make it happen.
+
+## Setting up your development environment
+
+If you haven't done so already, clone this repository and
+[follow the instructions in the README](https://github.com/pulumi/docs#pulumi-documentation-site)
+to set up your `pulumi/docs` development environment and run the development web server.
+
+Once you're able to run:
+
+```zsh
+$ make serve
+```
+
+... you can browse the site locally at http://localhost:1313/, and any changes you make to the pages of the site will be reloaded automatically in your browser.
+
+## How Learn content is organized
+
+All content lives under `./content/learn`. Modules, which appear as cards on the Learn home page, sit at the top level of the `learn` folder, and each of the topics that comprise a given module live as siblings underneath their parent module. There are only these two levels in the hierarchy; no further nesting is supported as of today.
+
+Both modules and topics can have their own meta images as well.
+
+```
+$ tree ./content/learn
+
+./content/learn
+├── _index.md
+└── pulumi-101-aws
+    ├── projects-and-stacks
+    │   ├── index.md
+    │   └── meta.png
+    ├── configuration-and-secrets
+    │   ├── index.md
+    │   └── meta.png
+    │   ...
+    ├── meta.png
+    └── _index.md
+```
+
+You can control the order in which modules appear on the Learn home page by adjusting the value of the `index` property in the module's YAML frontmatter:
+
+```
+---
+title: "Pulumi 101 on AWS"
+layout: module
+...
+
+# The order in which the module appears on the home page.
+index: 0
+...
+```
+
+Similarly, you can control the sort order of topics within a given module by adjusting their `index` properties as well:
+
+```
+---
+title: "Projects and Stacks"
+layout: topic
+...
+
+# The order in which the topic appears in the module.
+index: 0
+...
+```
+
+## Creating new modules or topics
+
+We have a couple of helpers that make it easy to create new modules and topics. Please use them! **Please don't copy and paste content from other modules or topics to create new ones.** If (er, when) you have trouble with one of the helpers, [file an issue](issues) so we can address it and make the authoring process better for everyone.
```

Comment:
It's going to be super tempting to copy and paste, especially for topics. Any reason topics won't work if we copy and paste?

Reply:
Not if you copy everything, no. But when you use the generator, it totally works, and you're sure to get a clean, correct jumping-off point. It's the same with the blog. Just helps prevent the kinds of issues that copy-paste always introduces.